README.md

```diff
@@ -5,6 +5,22 @@
 # libopenvizsla
 An attempt to reimplement [OpenVizsla](http://openvizsla.org/) host software in plain C.
 
+## Getting Started
+This section describes how to get started with OpenVizsla and Wireshark. \
+**IMPORTANT NOTICE:** As of late 2023, master is broken (see [#30](https://github.com/matwey/libopenvizsla/pull/30#issuecomment-1501915611)). Use Tag/Release 0.1.1 instead.
```

Comment:
Hi.

First of all, thank you for the contribution.

If you feel that something is broken, please open new issue instead.

Reply:
The current master gets out of sync and I do not really have time to debug it. As an intermediate solution, you can either use earlier master version or if you want to benefit from new bitstream use the original version of #30 (i.e. prior to master changes) that I pushed to a branch at my fork: https://github.com/desowin/libopenvizsla/tree/update-bitstream-working

Reply:
The bug mentioned here is fixed in #34